indexscroll/indexscroll/src/main/java/androidx/indexscroll/widget/SeslIndexScrollView.java

```diff
@@ -827,9 +827,14 @@ private void init() {
             mTextPaint.setAntiAlias(true);
 
             if (mSECRobotoLightRegularFont == null) {
-                mSECRobotoLightRegularFont = Typeface.create(mContext
-                        .getString(androidx.appcompat.R.string.sesl_font_family_regular),
-                        Typeface.NORMAL);
+                if (Build.VERSION.SDK_INT >= 34) {
```

Comment:
Salvo, can we lower to >= 28?
I used this api level on font logics for all other classes.
(note: 28 as it the lowest that supports font weight)